inginious/frontend/plugins/upcoming_tasks/__init__.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
```

Comment:
No need for shebang, this is not an executable file. The license header should be the same as for other files:

    # -*- coding: utf-8 -*-
    #
    # This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
    # more information about the licensing of this file.
    #
 
 You can still mention your name in the file as contributor afterwards , but the CLA (and, in your case, your employee contract :D) requires you to cede the copyright to UCLouvain, so this has to be mentioned.

Reply:
Well I believe I copy pasted the header from the auto-evaluation plugin actually... That I used as a base for the python file.
I replace it with what you just sent :)